drivers/node/ssh/ssh.go

```diff
@@ -675,6 +676,56 @@ func (s *SSH) SystemCheck(n node.Node, options node.ConnectionOpts) (string, err
 	return file, nil
 }
 
+//GetAvailableDrives returns the block drives on the node
```

Comment:
Can we change this method name? If I understand correctly this return all drives in a node. Can we use the name `GetDrives`

Reply:
Fixed